root/components/UserAccountTabs.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {CatalystContext} from '../context';
+import DBDefs from '../static/scripts/common/DBDefs';
+import {l} from '../static/scripts/common/i18n';
+
+import Tabs from './Tabs';
+
+const buildTab = (
+  page: string,
+  title: string,
+  path: string,
+  tabPage: string,
+) => (
+  <li className={tabPage === page ? 'sel' : null} key={tabPage}>
+    <a href={path}>{title}</a>
+  </li>
+);
+
+function buildTabs(
+  $c: CatalystContextT,
+  user: EditorT,
+  page: string,
+): React.Node {
+  const viewOwnProfile = $c.user && $c.user.id === user.id;
```

Comment:
Small nit, the old template had s/view/viewing/ and I think it makes more grammatical sense.

Reply:
fixed in https://github.com/metabrainz/musicbrainz-server/pull/718/commits/32cdf4950da69520edb47ffaa776e2383b35c7cb